Code/GraphMol/MolOps.h

```diff
@@ -850,69 +850,6 @@ RDKIT_GRAPHMOL_EXPORT std::list<int> getShortestPath(const ROMol &mol, int aid1,
 
 //@}
 
-#if 0
```

Comment:
Was it intentional to remove this chunk of code as part of this PR. I am asking just because it is not directly related, but I am happy with removing if you did it intentionally.

Reply:
yeah, it's not related to the PR, but it seems like it's always worth clearing out unused code. This particular bit has been disabled since we added the new canonicalization code 6 years ago